src/core/nativecall_libffi.c

```diff
@@ -777,7 +777,11 @@ MVMObject * MVM_nativecall_invoke(MVMThreadContext *tc, MVMObject *res_type,
     /* Free any memory that we need to. */
     if (free_strs)
         for (i = 0; i < num_strs; i++)
+#ifdef MVM_USE_MIMALLOC
+            free(ree_strs[i]);
+#else
```

Comment:
`ree_strs` - oops. (The compiler found this, not me. I happen to have been building libffi locally)

Reply:
Doh. A bad copy/paste that I noticed in the dynasm version, but not this one. Thanks